### DISCUSSION TITLE
Showcase page paragaon

### DISCUSSION THREAD

**Discussion by vitali-duckma** [openedx/paragon#1808] [Ideas]
Hi all, would be great to have a list of "who is using it" page on your website, or here on github.

**Comment by adamstankiewicz** [comment]
Hi @vitali-duckma! Apologies for the delayed response here. Given that the Paragon design system is created to support the Open edX platform, I'd be curious to hear your thoughts if the Paragon Github repo and/or documentation website linked to this existing ["Sites powered by Open edX Platform"](https://openedx.atlassian.net/wiki/spaces/COMM/pages/162245773/Sites+powered+by+Open+edX+Platform) page on the Open edX wiki?
